index.html

```diff
@@ -147,12 +155,39 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           purpose is to be handed to {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
         </p>
         <pre class="idl">
-          [Exposed = Window]
+          [Exposed = Window, Serializable]
           interface CropTarget {
             // Intentionally empty; just an opaque identifier.
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+
+        <p id="serializable-croptarget">
+          {{CropTarget}} objects are [= serializable objects =] [[!HTML]]. Their [= serialization
+          steps =], given <var>value</var> and <var>serialized</var>, are:
+        </p>
+        <ol>
+          <li>
+            Set <var>serialized</var>.[[\Data]] a randomly generated, globally unique {{DOMString}}.
+          </li>
+        </ol>
+        <p>
+          Their <a>deserialization steps</a>, given <var>serialized</var> and <var>value</var>, are:
+        </p>
+        <ol>
+          <li>
+            Compare <var>serialized</var>.[[\Data]] against a record of all {{CropTargets}} produced
+            and serialized thus far in the current top-level [=browsing context=], or in any of its
+            descendent [=browsing contexts=].
+            <ol>
+              <li>
+                If <var>serialized</var>.[[\Data]] was obtained by serializing previous
+                {{CropTarget}}, then set <var>value</var> to an equivalent {{CropTarget}}.
+              </li>
+              <li>Otherwise, set <var>value</var> to {{undefined}}.</li>
```

Comment:
If the serialisation steps are done correctly, we should always successfully deserialise, which does not seem to be the case here if we set value to undefined.
Is the intention to serialise a CropTarget and deserialise it into undefined?
Does this mean serialising a CropTarget from top level page A to top level page B then back to top level page A would not work?

Introducing a direct element <-> CropTarget link, for instance by having CropTarget having an element reference seems a good approach to tighten things.

Reply:
The `otherwise` section intended to deal with production of an invalid CropTarget. However, since we don't have a (legal) way to obtain an invalid CropTarget, I'd gladly drop this line - if that is indeed what you're suggesting...?